Templates/CSharp/EdgeDriverTemplate/packages.config

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="MicroBuild.Core" version="0.2.0" targetFramework="net46" developmentDependency="true" />
+  <package id="Microsoft.VSSDK.BuildTools" version="15.8.3247" targetFramework="net46" developmentDependency="true" />
+</packages>
```

Comment:
nit: Not a big deal but perhaps we can use `<PackageReference>` in the project file directly for these references?

Reply:
Fixed